device-types/HPE/Aruba-IAP-577.yaml

```diff
@@ -0,0 +1,40 @@
+---
+manufacturer: HPE
+model: Aruba IAP-577
+slug: hpe-aruba-iap-577
+part_number: R4H23A
+u_height: 0.0
+is_full_depth: false
+airflow: passive
+comments: '[Aruba 570 Series Datasheet](https://www.arubanetworks.com/assets/ds/DS_AP570Series.pdf)'
+weight: 2.1
+weight_unit: kg
+console-ports:
+  - name: Console Interface
+    type: usb-c
+    label: USBC
+interfaces:
+  - name: BLUETOOTH/Zigbee
```

Comment:
Is the interface name actually `BLUETOOTH / Zigbee`? The name here should match what you see on the CLI or programmatic interface if there is one